packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx

```diff
@@ -1213,6 +1224,21 @@ export const ComponentShowcase = () => {
             />
           ))}
         </CWTabsRow>
+
+        <CWText type="h3">Boxed Tabs</CWText>
+        <CWTabsRow boxed={true}>
```

Comment:
`boxed={true}` is redundant just the presence of `boxed` as a prop renders it truthy